lineman/app/components/navbar/navbar.scss

```diff
@@ -94,13 +94,22 @@
   text-align: center;
 }
 
-@media (max-width: 480px) {
+@media (max-width: 767px) {
+  .lmo-navbar__item {
+    display: inline-block;
+    text-align: center;
+    .lmo-navbar__btn {
```

Comment:
Do you need to be nesting this rule? If it was a sibling to .lmo-navbar__item it would have the same effect right?